db/chart-migrations/2021-11-03-schema-cleanup.sql

```diff
@@ -0,0 +1,243 @@
+update charts
+set config = JSON_REMOVE(config, '$."map"."isColorblind"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."isColorblind"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."variables"')
+where JSON_CONTAINS_PATH(config, 'one', '$."variables"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."map"."timelineMode"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."timelineMode"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."map"."minYear"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."minYear"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."chart-notes"')
+where JSON_CONTAINS_PATH(config, 'one', '$."chart-notes"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."iframe-height"')
+where JSON_CONTAINS_PATH(config, 'one', '$."iframe-height"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."/charts/255"')
+where JSON_CONTAINS_PATH(config, 'one', '$."/charts/255"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."iframe-width"')
+where JSON_CONTAINS_PATH(config, 'one', '$."iframe-width"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."activeLegendKeys"')
+where JSON_CONTAINS_PATH(config, 'one', '$."activeLegendKeys"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."chart-name"')
+where JSON_CONTAINS_PATH(config, 'one', '$."chart-name"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."map"."mode"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."mode"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."map"."targetYearMode"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."targetYearMode"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."useV2"')
+where JSON_CONTAINS_PATH(config, 'one', '$."useV2"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."yAxis"."labelDistance"')
+where JSON_CONTAINS_PATH(config, 'one', '$."yAxis"."labelDistance"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."map"."legendOrientation"')
+where JSON_CONTAINS_PATH(config, 'one', '$."map"."legendOrientation"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."/charts/29"')
+where JSON_CONTAINS_PATH(config, 'one', '$."/charts/29"') = 1;
+
+update charts
+set config = JSON_REMOVE(config, '$."isExplorable"')
+where JSON_CONTAINS_PATH(config, 'one', '$."isExplorable"') = 1;
```

Comment:
This one still exists in corners of the codebase unfortunately ([example 1](https://github.com/owid/owid-grapher/blob/da1f036a24cbfd83b85348e31456fa5a32211f78/adminSiteServer/apiRouter.ts#L223-L235), [example 2](https://github.com/owid/owid-grapher/blob/2732ac19a1946369c7d34c3a5562076605bc2440/db/model/Chart.ts#L32)), but we should get rid of it, because it's no longer has any effect on the chart (it used to)